bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -170,7 +170,7 @@ metadata:
       Cloud Provider,Developer Tools,Modernization & Migration,OpenShift Optional,Storage
     certified: "false"
     containerImage: quay.io/konveyor/oadp-operator:latest
-    createdAt: "2024-05-29T17:22:16Z"
+    createdAt: "2020-09-08T12:21:00Z"
```

Comment:
NIT: this should not be changed by make bundle (but it is ok, wont break anything)

Reply:
well.. it is a change.. it is a valid change, it is a new creation of CSV :)

Unless this is the only change.. it is technically a new CSV.

Reply:
agreed on the adjustment due to valid change, but... createdAt **2020**?? :laughing:

Reply:
At least have it close to `2024-06-20`, no?

Reply:
oh I though it was updated from the command, but I think tiger just copied what was our create at (same as in oadp-1.3 branch for example)

would be good practice to update once in a while (every release?), but would discuss a little how/when change the date

Reply:
good catch.. with the date..

Reply:
I think how/when to update should happen whenever there is a legitimate change to CSV -- as is the case here with the RELATED_IMAGE update.

Bumping `createdAt` for the sake of updating the timestamp with no other changes creates additional burden downstream reconciling against the CSV because we look at its digest and fail the build if it doesn't match what we expect.

Reply:
Updating  `createdAt:` value at Z-stream intervals (with no other accompany CSV changes) is not unreasonable.  If that is something desired upstream, I won't fight it, but it would introduce additional effort, however nominal, downstream that does not currently exist. :sunglasses:

Reply:
will try to think a way to update createdAt always there is change in CSV, and enforce CI to not let just the date change without any other changes 

also enforce date to be higher than previous one heheheh